File: app/src/release/java/com/mycompany/myapp/monitoring/model/NoOpTree.java
package com.mycompany.myapp.monitoring.model;

import timber.log.Timber;

public class NoOpTree extends Timber.Tree {
    @Override
    protected void log(int priority, String tag, String message, Throwable t) {

    }

    @Override
    protected boolean isLoggable(String tag, int priority) {
        return false;
    }
}

File: app/src/release/java/com/mycompany/myapp/monitoring/model/NoOpTree.kt
package com.mycompany.myapp.monitoring.model

import timber.log.Timber

class NoOpTree : Timber.Tree() {
    override fun log(priority: Int, tag: String, message: String, t: Throwable) { }

    override fun isLoggable(tag: String, priority: Int) = false
}
